packages/fxa-auth-server/lib/senders/templates/index.js

```diff
@@ -4,93 +4,115 @@
 
 'use strict';
 
+const LIB_DIR = '../..';
```

Comment:
This is just the same `index.js` that was previously in the `subscription-templates` directory. But because there was an old `index.js` here before, it's showing as a diff.